data/data-samples/sample-data-oracle-reactive-book/src/main/java/io/americanexpress/data/oracle/book/DataBookApplication.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.americanexpress.data.oracle.book;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * {@code DataBookApplication} DataBookApplication sample is to show how to use synapse-oracle-data
+ */
+@SpringBootApplication
+public class DataBookApplication {
```

Comment:
```suggestion
public class BookDataApplication {
```

Reply:
please remove the application class from the data module